mateo-mustapic/assignments/liferay-app/sdk/portlets/todo-portlet/docroot/view.jsp

```diff
@@ -25,12 +25,18 @@
 		<i class="icon-check"></i>
 	</h3>
 
+	<portlet:resourceURL var="resourceUrl"></portlet:resourceURL>
+
 	<aui:form cssClass="task-form" name="task-form">
-		<aui:input cssClass="input-item" name="task" placeholder="enter-task" required="true" />
+		<aui:input cssClass="input-item" data-resourceURL="<%= resourceUrl.toString() %>" name="task" placeholder="enter-task" required="true" />
```

Comment:
Instead of passing the URL in data attribute, pass it in JS module config. The only case when you would want to do it in data attribute is if the node is created in a loop and you need to access a specific node relative to click later on.